build.gradle

```diff
@@ -262,6 +288,18 @@ dependencies {
   functionalTestCompile sourceSets.main.runtimeClasspath
 
   smokeTestCompile sourceSets.main.runtimeClasspath
+
```

Comment:
can we please validate if it requires all these Lombok elements Or may be research whats is its purpose.